.kitchen.yml

```diff
@@ -32,4 +32,4 @@ suites:
       command_timeout: 1800
       root_module_directory: test/fixtures/full
       # setting version verification to false since it requires TF to be less than v1.1
-      verify_version: false
\ No newline at end of file
+      verify_version: false
```

Comment:
lets delete this file since it is not used

Reply:
done